### PR TITLE
Put people going through the sec chute to "Released" status

### PR DESCRIPTION
**Pull request by BotchKing**
[A-Game-Objects][C-QoL][size/XS]

## About the PR 
The flusher already removed cuffs, so now it will also set your criminal record to "Released"

## Why's this needed? 
People often throw criminals in the brig without clearing their arrest status, leading to them being wrongfully arrested again later. This should make things faster and hopefully prevent unnecessary arrests due to miscommunication or laziness.

## Changelog
```changelog
(u)BotchKing
(+)Changes criminal status to "released" upon being flushed
```

### CODE REVIEW

**Comment by NightmareChamillian** [comment]
changelog is borked up, make sure to have the triple `  signs, as well as what the entry is for the changelog

**Comment by Chatauscours** [comment]
Ideally, I would prefer if it was setting them to released only if they are set either to Incarcerated or arrest, that way if the guy is already set to parolled or not set at all (not changing the status because it's a joke crime for example), it won't ovveride it.

File: code/obj/machinery/floorflusher.dm [hunk 1]
```diff
@@ -228,6 +228,11 @@
 					boutput(M, "You feel your handcuffs being removed.")
 					M.handcuffs.drop_handcuffs(M)
 
+				//Might as well set their security record to "released"
+				SPAWN(1)
```
Comment (amylizzle) [inline code/obj/machinery/floorflusher.dm:232]: Why is this in a SPAWN?

Comment (BotchKing) [reply code/obj/machinery/floorflusher.dm:232]: Long story short: I wanted to make sure the process wasn't causing enough lag to mess up the disposal work. In testing I've had crashes due to flusher sending me in the void when destroyed by flipping and wiggling. I realized this issue is in the main branch, however, so the SPAWN is no longer necessary indeed

Comment (ZeWaka) [reply code/obj/machinery/floorflusher.dm:232]: i'd remove it then

Comment (BotchKing) [reply code/obj/machinery/floorflusher.dm]: Done, I also added a condition so only people on arrest or or incarcerated get the released status

File: code/obj/machinery/floorflusher.dm [hunk 2]
```diff
@@ -228,6 +228,10 @@
 					boutput(M, "You feel your handcuffs being removed.")
 					M.handcuffs.drop_handcuffs(M)
 
+				//Might as well set their security record to "released"
+				var/datum/db_record/R = data_core.security.find_record("name", M.name)
+				if(!isnull(R) && ((R["criminal"]=="Incarcerated") || (R["criminal"]=="*Arrest*")))
+					R["criminal"]="Released"
```
Comment (ZeWaka) [inline code/obj/machinery/floorflusher.dm:234]:
```suggestion
				if(!isnull(R) && ((R["criminal"] == "Incarcerated") || (R["criminal"] == "*Arrest*")))
					R["criminal"] = "Released"
```
code guide - see: style guidelines

**Review by ZeWaka — APPROVED** [review]

**Comment by ZeWaka** [comment]
thanks for the PR!

**Pull request merged**